packages/block-library/src/quote/edit.js

```diff
@@ -43,40 +58,17 @@ export default function QuoteEdit( {
 				/>
 			</BlockControls>
 			<BlockQuotation { ...blockProps }>
-				<RichText
-					identifier="value"
-					multiline
-					value={ value }
-					onChange={ ( nextValue ) =>
-						setAttributes( {
-							value: nextValue,
-						} )
-					}
-					onMerge={ mergeBlocks }
-					onRemove={ ( forward ) => {
-						const hasEmptyCitation =
-							! citation || citation.length === 0;
-						if ( ! forward && hasEmptyCitation ) {
-							onReplace( [] );
-						}
-					} }
-					placeholder={
-						// translators: placeholder text used for the quote
-						__( 'Write quote…' )
-					}
-					onReplace={ onReplace }
-					onSplit={ ( piece ) =>
-						createBlock( 'core/quote', {
-							...attributes,
-							value: piece,
-						} )
-					}
-					__unstableOnSplitMiddle={ () =>
-						createBlock( 'core/paragraph' )
-					}
-					textAlign={ align }
+				<InnerBlocks
```

Comment:
It would be good to try the new `__experimentalInnerBlocks` API here (#25633).

Reply:
This would eliminate an extra wrapper div and then the markup would be exactly the same as the front-end.

Reply:
Ah, excellent! I was wondering how would I do that.